wsd/DocumentBroker.cpp

```diff
@@ -5302,17 +5302,23 @@ void DocumentBroker::dumpState(std::ostream& os)
     else
         os << " none";
 
-    os << '\n';
-    _lockCtx->dumpState(os);
+    if (_lockCtx)
+    {
+        os << '\n';
+        _lockCtx->dumpState(os);
+    }
 
     if (_tileCache)
     {
         os << '\n';
         _tileCache->dumpState(os);
     }
 
-    os << '\n';
-    _poll->dumpState(os);
+    if (_poll)
```

Comment:
coverity points out that in dumpState we have already called getIOStats before this and that calls ASSERT_CORRECT_THREAD() aka DocumentBroker::assertCorrectThread() which  dereferences _poll. Seeing as we create _poll in the ctor and never reset it we could presumably drop this check.

Reply:
Fair point. I added it for consistency, but agreed that it's unnecessary.